src/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+/* #region childs */
+.stars--1 > :first-child,
+.stars--2 > :first-child,
+.stars--3 > :first-child,
+.stars--4 > :first-child,
+.stars--5 > :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > :nth-child(2),
+.stars--3 > :nth-child(2),
+.stars--4 > :nth-child(2),
+.stars--5 > :nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > :nth-child(3),
+.stars--4 > :nth-child(3),
+.stars--5 > :nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > :nth-child(4),
+.stars--5 > :nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > :last-child {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)